src/components/CharacterSelector.vue

```diff
@@ -252,7 +245,7 @@ function focusLastButtonInSection(section) {
 
 .tap-characters--dropdown {
   width: fit-content;
-  padding: var(--spacing-2);
+  padding: var(--spacing-4);
```

Comment:
This is now `var(--spacing-2)` just like before, with the additional `var(--dropdown-padding)`, which also equals `var(--spacing-2)`.